README.md

```diff
@@ -132,7 +132,7 @@ VIP Workflow adds a "Editorial Metadata" section to the post sidebar, which allo
 
 ### Editorial Metadata
 
-A post type that's supported at first, and then unsupported is used to make posts, then those posts will still show editorial metadata fields.
+If a post is created as a type that supports Editorial Metadata, that metadata will remain even if the post type is modified not to support Editorial Metadata. In other words, once a post is created with Editorial Metadata fields, those fields will remain regardless of configuration changes.
```

Comment:
@ingeniumed I'm not sure if I have this right, but I thought I would take a stab.

Reply:
Also, we don't include any information about what post types will have editorial metadata or how to change that. I would guess it is the same as the post types that include custom statuses, but that isn't explicit.

Reply:
This is great, I like the way that it's been phrased